docs/software/camera.md

```diff
@@ -57,6 +57,15 @@ It consists of full control of the camera in a web-based interface.
     dietpi-software reinstall 59
     ```
 
+=== "RPi camera module"
+
+    If you have an official Raspberry Pi camera module, it should be activated by the installation process. This can be checked/enabled via `dietpi-config` > `Display Options` > `RPi Camera` to show `[On]`.  
```

Comment:
```
If you have an official Raspberry Pi camera module, it should be activated by the installation process.
```
=>
```
The RPi camera module is automatically enabled during the installation process. It requires however a reboot and/or sometimes a power cycle to take effect.
```
Since RPi Cam Control is for the RPi camera module explicitly, it is always enabled, regardless whether "you have an official Raspberry Pi camera module". And let's use absolute wording here ("is" vs "should be"): This must work, else it is a bug that should be reported. The hint with the required reboot and in case power cycle of course is still helpful.